packages/atomic/cypress/integration/facets/category-facet/category-facet.cypress.ts

```diff
@@ -133,7 +133,7 @@ describe('Category Facet Test Suites', () => {
       describe('when selecting the "Show more" button', () => {
         function setupShowMore() {
           setupGoDeeperOneLevel();
-          pressShowMore();
+          pressShowMore(CategoryFacetSelectors);
```

Comment:
Some of these actions are now part of `facet-common-actions`, so they need to take a selector as a parameter.